src/compas_ags/rhino/forceartist.py

```diff
@@ -101,7 +101,7 @@ def draw(self):
             dual_edges = list(self.diagram.dual.edges())
             for index, edge in enumerate(self.diagram.ordered_edges(self.diagram.dual)):
                 f = self.diagram.dual.edge_attribute(dual_edges[index], 'f')
-                text[edge] = "%s kN {%s}" % (round(f, 2), index)
+                text[edge] = "%s kN {%s}" % (round(abs(f), 2), index)
```

Comment:
is 1 digit not enough?
also, we should introduce a system of units that allows us to omit digits all together, but that is for a different PR...